apiserver/Makefile

```diff
@@ -32,6 +32,14 @@ jupyter_notebook_config.py:
 	    -e "s,@keyfile@,$(GSA_KEY_FILE),g" \
             < jupyter_notebook_config.py.in > $@
 
+hailjwt: ../hailjwt/setup.py $(find ../hailjwt/hailjwt -iname \*.py)
+	-mkdir hailjwt
+	cp ../hailjwt/setup.py hailjwt/
+	rsync -a --prune-empty-dirs --include '*/' --include '*.py' --exclude '*' ../hailjwt/hailjwt/ hailjwt/
+
+install-hailjwt: hailjwt
```

Comment:
I would prefer we don't modify the local environment during tests.  Can we just set PYTHONPATH to include ../hailjwt/hailjwt?

Reply:
I basically only wanted to copy other packages for docker to avoid the large context problem.

Reply:
done